### PR TITLE
[Test] Test disconnected upgrade from RHCS 7.x to RHCS 8

### PR DESCRIPTION
**Pull request by sayaleeraut**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by MohitBis** [comment]
@sayaleeraut 

Can you please add passing logs?

File: tests/ceph_installer/test_disconnected_upgrade.py [hunk 1]
```diff
@@ -12,15 +46,108 @@ def run(ceph_cluster, **kw):
...
+        )
+
+    # Setup ssh key to all new nodes
+    for _node in _nodes:
+        setup_ssh_keys(_node, nodes)
+
+    # Login to the registry using podman command on all the new nodes
+    private_registry_url = f"{installer.hostname}:5000"
+    for _node in _nodes:
+        cmd = f"podman login {private_registry_url} -u {PRIVATE_REG_USER} -p {PRIVATE_REG_PASS} --tls-verify=False"
```
Comment (vamahaja) [inline tests/ceph_installer/test_disconnected_upgrade.py:122]: Use - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py#L9

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: tests/ceph_installer/test_disconnected_upgrade.py (hunk 1 above)
Comment (sayaleeraut) [reply tests/ceph_installer/test_disconnected_upgrade.py]: done, please review @vamahaja

**Comment by sayaleeraut** [comment]
Test execution logs -http://magna002.ceph.redhat.com/ceph/cephci-jenkins/saraut/PR_4205/ 

Please note, since RHCS8 repo and container images are not live yet, I manually added the 2 and rest part of upgrade was done by automation script itself.

**Review by MohitBis — APPROVED** [review]
LGTM

**Pull request merged**